docs/proposals/RuntimeValueWitness.rst

```diff
@@ -58,6 +58,7 @@ copied, but do not need to be released or retained.
    I16 = 's',
```

Comment:
This whole file is outdated and should be removed/replaced with the current design